### ISSUE TITLE
Confused by rebase scenario

### ISSUE THREAD

**Issue by philipstaffordwood** [git-learning-game/oh-my-git#65]
This is a super useful way of teaching, 
specifically with `git`. I've been thinking of doing something similar for a while...

I have to say:
This is an **AWESOME** execution of the idea!

I'm confused by the **rebase** scenario:

As far as I can make out the intent is something like:
After:
```
git checkout donut      # start in donut
git rebase coffee       # suggested action by wording of 
                        # "it will copy the events in 
                        #  your current timeline after 
                        #  the specified one"
get checkout main
git reset --hard donut
```

It's not green:

```true
{ git show main:you | grep "You ate.*baguette"; }    # false
                                                     # is "You do not have a baguette."
```
because the first commit to the **coffee** branch (branched off the **baguette** branch) overwrites the eaten baguette.

https://github.com/git-learning-game/oh-my-git/blob/afef5aa5b6a3277f3ec95b2fac7b1acf688f6152/levels/changing-the-past/rebase#L41

```
   && 
    { git show main:you | grep "You drank.*coffee"; }  # true
   && 
    { git show main:you | grep "You ate.*donut"; }     # true
   && 
   { test "$(git log main --oneline | wc -l)" -eq 7; } # true
```

fixing `"You do not have a baguette."` with a commit  means that:

```
   { test "$(git log main --oneline | wc -l)" -eq 7; } # false
```
fails since there are now more commits.

**Am I  missing something?**

**Is there another intended approach?**

**Comment by TrevorBenson** [comment]
I found this as well juts a few minutes ago. I wasn't sure if `coffee` branch was intended to branch from `baguette` or `main`. I was going to check if that also ends up at 7 steps. Looking to see how to configure game/pak to use an alternative repo/branch for testing. 

Otherwise setting line 40 to `git checkout -b coffee main` should give 3 unique branches to bring together, which if 7 commands should result in the same success for the test.

**Comment by TrevorBenson** [comment]
Read README.md, tested against my fork. Making `coffee` branch from `main` allows all requirements of test to complete within 7 commands. PR following.

**Comment by blinry** [comment]
Thanks for your kind words, @philipstaffordwood, really happy to hear that! :) This problem should be fixed by #52!
